.github/workflows/check.yaml

```diff
@@ -116,12 +116,21 @@ jobs:
             echo "TEST_MODEL_CONSTRAINTS=arch=arm64" >> "$GITHUB_ENV"
           fi
 
+      - name: Build the charm
+        run: charmcraft -v pack
+
       - name: Run tests
-        run: ${{ matrix.test-command }}
+        run: |
+          export CHARM_PATH_JAMMY="$(pwd)/$(ls | grep '.*22.04.*\.charm$')"
+          echo "$CHARM_PATH_JAMMY"
+          export CHARM_PATH_FOCAL="$(pwd)/$(ls | grep '.*20.04.*\.charm$')"
+          echo "$CHARM_PATH_FOCAL"
+          ${{ matrix.test-command }}
```

Comment:
non-blocker: seems that the file generated is the same for both. Something like:

```
local-users_ubuntu-22.04-amd64_ubuntu-20.04-amd64.charm
```

Can't we use a single charm path variable for both?

Reply:
Please see comment here: https://github.com/canonical/charm-userdir-ldap/pull/47#discussion_r1764142777